test/test_data/circllhist_proto_json.gold

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
I wonder if we could somehow unify this with the hdr histogram and have a single gold file / avoid having two different ones.. but probably we're going to see minor differences because of the 2-based vs 10-based implementation details, right?

Reply:
Yes, since the 2 histogram implementation has different precision, the values in the gold file will be different.